JS/Lesson32/savas/my-app/src/App.css

```diff
@@ -0,0 +1,3 @@
+div.recipe-list{
```

Comment:
Since this code is related to the RecipeList component, it's better to add a `styles.css` file in the RecipeList folder and import it to the `RecipeList/index.jsx` file.  This way you project will be cleaner and easier to maintain in long term.

Reply:
Anna, this is actually not a style related to RecipeList. I want to use it to put the recipe list and details side by side. Thus, I think to change the className as "content". Is it appropriate?

Reply:
I see now. It would be better to use a name "container" :)

Reply:
Actually, the parent div's class is "container", so I put the name as "content". In that case, is "sub-container" appropirate?

Reply:
I'd say sub-container is not a common name to use but let it be in this case :)

Reply:
Ok, thank you :)